### DISCUSSION TITLE
Help with UI just a bit

### DISCUSSION THREAD

**Discussion by taslabs-net** [tobychui/zoraxy#285] [General]
Hello! First let me start by THANKING YOU @tobychui for this entire effort.  I work for a large CDN company and I love showing Zoraxy on calls.  Please message me for details if you'd like.  I LOVE THIS PRODUCT.

Okay, now the question. I see other screen shots where it shows info like the host etc. I see none of that.  I have it installed using the amazing helper scripts.  And secondly, is there a way to get better vision on the error requests? 

![image](https://github.com/user-attachments/assets/13b4c40a-7c6a-48f9-bbf1-c9141998e1f9)

**Comment by tobychui** [comment]
Hi @taslabs-net,

Thank you so much for your enthusiasm for Zoraxy! As I can't find your contacts on your Github homepage, you can reach me via email at toby@imuslab.com for private messages.

Regarding your questions, 

> I see other screen shots where it shows info like the host etc. I see none of that.

The router type summary is based on what service you are trying to reverse proxy. If you are only hosting static websites (e.g. with no websocket or virtual directories), this is perfectly normal to only see Host (http) in the summary.

> is there a way to get better vision on the error requests?

You can find the log viewer in the utilities page and filter the log based on error / request types. 
![圖片](https://github.com/user-attachments/assets/b6d4832b-38a2-4979-993a-6eb7f749d14c)

![圖片](https://github.com/user-attachments/assets/9a24bb84-7e21-45b2-af06-619b4790f2f2)

If your log is huge (as I assume seeing you have 434k request per day), you might want to download the log and use some external log analysis tool for checking what request error it is. The Zoraxy log is partially compatible with Apache log so it should not be too hard to find a log viewer that can read it.

**Reply by taslabs-net** [reply]
Amazing work should be celebrated! I'll send you a note.  

434k was just a small test haah! I appreciate the explanations and that makes sense.  I hate logging/metrics as I'm so bad at connectors and sensors and something else.  Anyways keep it up!!  This is badass.
